VDA-SBT/test/index.ts

```diff
@@ -347,12 +347,12 @@ describe("Verida Soulbound", () => {
 
     describe("Get Claimed SBT list", () => {
         it("Should return tokenId list for the claimer", async () => {
-            const idList = await contract.connect(claimer).getClaimedSBTList()
```

Comment:
Before, `getClaimedSBTLIst()` function returns the list of caller. So I had to call the `contract.connect(claimer)`.
Now, no need to call the `getClaimedSBTList()` function from the claimer. So removed the `connect(...)`.